apps/i18n/common/en_us.json

```diff
@@ -1809,6 +1815,7 @@
   "restoreSection": "Restore Section",
   "restore": "Restore",
   "restoreThisVersion": "Restore this Version",
+  "restrictedShareInfo": "Publish and Remix are disabled on this project because you have chosen to upload images. You can undo this change with Version History.",
```

Comment:
nit: since we say "use Version History" above, should we say "using Version History" here?